sycl/include/CL/sycl/INTEL/dot_product.hpp

```diff
@@ -0,0 +1,76 @@
+//==----------- dot_product.hpp ------- SYCL dot-product -------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// DP4A extension
+
+#pragma once
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace intel {
```

Comment:
As for my understanding the namespaces scheme has changed (according to release notes: https://github.com/intel/llvm/blob/sycl/sycl/ReleaseNotes.md):
 - Moved hardware targeted extensions to INTEL namespace [3084982]
 - Moved general language extensions to ONEAPI namespace [a73369d]

I assume that we should not use cl::sycl::INTEL namespace anymore

Reply:
OK, moved to ONEAPI namespace.